applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application_variables.cpp

```diff
@@ -11,12 +11,20 @@ KRATOS_CREATE_VARIABLE(double, ADJOINT_VELOCITY_POTENTIAL)
 KRATOS_CREATE_VARIABLE(double, ADJOINT_AUXILIARY_VELOCITY_POTENTIAL)
 
 // Flow field magnitudes
-KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_INFINITY)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_LOWER)
 KRATOS_CREATE_VARIABLE(double, PRESSURE_LOWER)
 KRATOS_CREATE_VARIABLE(double, POTENTIAL_JUMP)
 KRATOS_CREATE_VARIABLE(double, ENERGY_NORM_REFERENCE)
 KRATOS_CREATE_VARIABLE(double, POTENTIAL_ENERGY_REFERENCE)
+KRATOS_CREATE_VARIABLE(double, CP)
+KRATOS_CREATE_VARIABLE(double, GAMMA)
```

Comment:
What is this gamma? Please use more self-explicative names.

Reply:
It's the [Heat capacity ratio](https://en.wikipedia.org/wiki/Heat_capacity_ratio) (or adiabatic index). 
We defined it as the greek letter (gamma), but I understand that it's better to specifically define the variables.
We can change it to HEAT_CAPACITY_RATIO or ADIABATIC_INDEX. Would it make sense to also add it in the cfd_variables?

Reply:
I personally like more heat_capacity_ratio, I will change it to that one for now and then we can decide whether another name is suitable and whether it makes sense to move it to the cfd_variables.